catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/handlers/CqlTransformHandler.java

```diff
@@ -38,8 +38,9 @@
 import org.apache.tika.mime.MimeTypeException;
 import org.apache.tika.mime.MimeTypes;
 import org.codice.ddf.catalog.ui.metacard.transform.CsvTransform;
-import org.codice.ddf.catalog.ui.query.cql.CqlQueryResponse;
-import org.codice.ddf.catalog.ui.query.cql.CqlRequest;
+import org.codice.ddf.catalog.ui.query.utility.CqlQueryResponse;
+import org.codice.ddf.catalog.ui.query.utility.CqlRequest;
+import org.codice.ddf.catalog.ui.util.CqlQueryUtil;
 import org.codice.ddf.catalog.ui.util.EndpointUtil;
```

Comment:
Do we still need EndpointUtil in this class?

Reply:
yeah other function from endpointUtil are used https://github.com/codice/ddf/pull/5806/files/1777c3ac350e61be0512a170ec862bb5b8ba86e3#diff-cc5f97af83867437776fddd13f08d36bR124